tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -282,7 +281,7 @@ public:
          value.GetField()->Append(value);
       }
       fNEntries++;
-      if ((fNEntries % fClusterSizeEntries) == 0)
```

Comment:
Maybe the `fClusterSizeEntries` data member is unused now?  I would either remove it or initialize it in the `RNTupleWriter` constructor using the value of `fSink->GetWriteOptions().GetNClusterEntries()`.

Reply:
Thank you Javier, done in 6930897.